### DISCUSSION TITLE
Combobox in docs doesn't behave correctly

### DISCUSSION THREAD

**Discussion by veloware** [chakra-ui/ark#2504] [General]
### Description

When I choose an option in the combobox and then try and change to another option I should be able to add another selected option, but I cannot select any option other than the already selected option - 

https://ark-ui.com/react/docs/components/combobox

![image](https://github.com/chakra-ui/ark/assets/61578546/3653aa71-4cf8-49fe-842f-ee9585cc1d39)

### Link to Reproduction (or Detailed Explanation)

https://ark-ui.com/react/docs/components/combobox

### Steps to Reproduce

Go to https://ark-ui.com/react/docs/components/combobox
Type 'react'
Select 'react'
Clear the input
Type 'vue'
Notice 'vue' is not appearing as an option, only 'React' appears

### Ark UI Version

current docs

### Framework

- [X] React
- [ ] Solid
- [ ] Vue

### Browser

Chrome

**Comment by segunadebayo** [comment]
This is now fixed.
